authors/apps/articles/models.py

```diff
@@ -57,3 +66,42 @@ class Rating(models.Model):
 
     def __str__(self):
         return self.user_rating
+
+
+@receiver(post_save, sender=Article)
+def send_notifications_to_all_users(sender,
+                                    instance,
+                                    created, *args, **kwargs):
+    """Create a Signal that sends email to all users that follow the author.
+     Arguments:
+        sender {[type]} -- [Instance of ]
+        created {[type]} -- [If the article is posted.]
+    """
+
+    if instance and created:
+        users_followers = instance.author.followers.all()
+
+        link = f'{os.getenv("HEROKU_BACKEND_URL")}/api/v1/articles/' + \
```

Comment:
You can use triple quotes on f-strings to use them over multiple lines. Check: https://www.reddit.com/r/Python/comments/7kajpm/how_to_do_multiline_f_string_without_the/

Reply:
Done pushing changes right now